src/test/test-execute.c

```diff
@@ -154,6 +155,19 @@ static void test_exec_runtimedirectory(Manager *m) {
                 test(m, "exec-runtimedirectory-owner.service", 0, CLD_EXITED);
 }
 
+static void test_exec_capabilityboundingset(Manager *m) {
+        /* We use capsh to test if the capabilities are
+         * properly set, so be sure that it exists */
+        if (find_binary("capsh", NULL) != 0)
+                return;
```

Comment:
Is it silently skip test?
Maybe, `log_*`?
Something like [that](https://github.com/systemd/systemd/blob/master/src/test/test-capability.c#L73):
```
        if (!nobody) {
                log_error_errno(errno, "Could not find nobody user: %m");
                return -EXIT_TEST_SKIP;
        }
```